packages/rlc-common/src/helpers/nameConstructors.ts

```diff
@@ -108,3 +108,16 @@ export function getImportModuleName(name: ModuleName, codeModel: RLCModel) {
 
   return name.cjsName;
 }
+
+export function getClientName(model: RLCModel) {
+  const clientName = model.libraryName;
+  let clientInterfaceName = clientName.endsWith("Client")
+    ? `${clientName}`
+    : `${clientName}Client`;
+
+  if (model.options?.isModularLibrary) {
+    clientInterfaceName = `${clientName.replace("Client", "")}Context`;
```

Comment:
nitpick: is this equivalent?
```suggestion
    clientInterfaceName = clientName.replace("Client", "Context");
```